node-process-argv/echo.js

```diff
@@ -0,0 +1,7 @@
+// not sure if user can type more than one message?
+//
+// const [, , ...message] = process.argv;
+// console.log(message.join(' '));
```

Comment:
Nice! This would correctly log all the arguments. It works correctly if there is just one argument, too.